packages/safe-ds-lang/src/language/safe-ds-module.ts

```diff
@@ -204,13 +216,17 @@ export const createSafeDsServices = function (context: DefaultSharedModuleContex
  *  - Services specified in this file
  *
  * @param context Optional module context with the LSP connection.
+ * @param runnerCommand Optional command to start the runner, if available.
  * @return An object wrapping the shared services and the language-specific services.
  */
-export const createSafeDsServicesWithBuiltins = async function (context: DefaultSharedModuleContext): Promise<{
+export const createSafeDsServicesWithBuiltins = async function (
+    context: DefaultSharedModuleContext,
+    runnerCommand: string | undefined = undefined,
```

Comment:
For the sake of future extensibility, I'd suggest wrapping the `runnerCommand` into an options object:

```ts
export const createSafeDsServicesWithBuiltins = async function (
    context: DefaultSharedModuleContext,
    options?: ModuleOptions
) // ...

interface ModuleOptions {
    runnerCommand?: string
}
```